front/app/containers/InitiativesShow/ReactionControl/ProposedNotReacted.tsx

```diff
@@ -166,22 +149,19 @@ const ProposedNotReacted = ({
   initiative,
   initiativeSettings: { reacting_threshold, threshold_reached_message },
   disabledReason,
-}: Props) => {
+}: StatusComponentProps) => {
   const theme = useTheme();
   const reactionCount = initiative.attributes.likes_count;
   const reactionLimit = reacting_threshold;
-  const daysLeft = getPeriodRemainingUntil(initiative.attributes.expires_at);
 
-  const thresholdReachedTooltip = threshold_reached_message ? (
+  const thresholdReachedTooltip = (
     <IconTooltip
```

Comment:
Just want to double check - removing the check for `threshold_reached_message` here was intended?